src/interactivity/win32/WindowMetrics.cpp

```diff
@@ -170,9 +164,9 @@ RECT WindowMetrics::GetMaxWindowRectInPixels(const RECT* const prcSuggested, _Ou
 // - dwExStyle - Extended Style flags
 // Return Value:
 // - BOOL if adjustment was successful. (See AdjustWindowRectEx definition for details).
-BOOL WindowMetrics::AdjustWindowRectEx(_Inout_ LPRECT prc, const DWORD dwStyle, const BOOL fMenu, const DWORD dwExStyle)
+BOOL WindowMetrics::AdjustWindowRectEx(_Inout_ til::rect* prc, const DWORD dwStyle, const BOOL fMenu, const DWORD dwExStyle)
 {
-    return ServiceLocator::LocateHighDpiApi<WindowDpiApi>()->AdjustWindowRectExForDpi(prc, dwStyle, fMenu, dwExStyle, ServiceLocator::LocateGlobals().dpi);

```

Comment:
>ServiceLocator::LocateHighDpiApi<WindowDpiApi>()-> [](http://example.com/codeflow?start=11&length=50)

note to self: yea this is safe to remove, we're already in the win32 interactivity, this can call user32 apis just fine

Reply:
This calls the other `AdjustWindowRectExForDpi` in the same file actually. I simply deduplicated that line of code so that we only need one `->as_win32_rect()`.

Reply:
Interesting. I thought we used LocateHighDpiApi for _compatibility with older versions of Windows_, not OneCore. We have both kinds of delay loaders unfortunately.

Reply:
Oh that could be right. That kinda looks like it's only used to avoid using the that API when building in OSS...? That's a weird one.